### DISCUSSION TITLE
data.json für Amazon music

### DISCUSSION THREAD

**Discussion by Ozelotschwein** [Thyraz/Sonos-Kids-Controller#21] [Ideas]
Here the data.json for: 

TKKG 1-100
(without 19+20)
[TKKG_1_100 .txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/5848863/TKKG_1_100.txt)
Status: will be continued

"Die drei??? Kids" Folge 1-80
(80 is not published yet)
[fragezeichen_Kids_1_80.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/5830712/fragezeichen_Kids_1_80.txt)

Die drei ??? 
Status: planned

Benjamin Blümchen
[Benjamin 100_147.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/5851364/Benjamin.100_147.txt)
Status: done

Bibi & Tina 1-49
[Bibi-Tina1-94.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/8504753/Bibi-Tina1-94.txt)

HerrH
[herrH.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/5851462/herrH.txt)

Schleich Horse Club Folge 1-14
[Schleich_Horse_Club.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/5956819/Schleich_Horse_Club.txt)
Status: done

TKKGjunior 1-21
[TKKGjunior.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/8504733/TKKGjunior.txt)

**Comment by NoTrace-Kai** [comment]
Hi ...

Super ... vielen Dank (Thanks a lot)
Did someone wrote the data.json for "Bibi und Tina" or "Bibi Blocksberg"?

CU Kai

**Reply by Ozelotschwein** [reply]
Hello Kai,
die drei ??? Kids werde ich noch fertig stellen danach plane ich TKKG bis ende der Woche

Gruß Matthias

**Reply by NoTrace-Kai** [reply]
Dann mal viel Erfolg ;) Freu mich schon auf Deine Posts hier !

**Comment by rumpelfish** [comment]
hi here's Yakari from Apple Music

[yakari_data_json.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/5832099/yakari_data_json.txt)

**Reply by neomorph77** [reply]
Sehr gut, Danke!!!!

**Reply by NoTrace-Kai** [reply]
Hab auch noch mal etwas gearbeitet ... AppleMusic -
- Die Garde der Löwen 1-12
- Der König der Löwen 1-3 

[die garde und der könig der löwen.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/5965828/die.garde.und.der.konig.der.lowen.txt)

**Comment by NoTrace-Kai** [comment]
Hat sich denn noch niemand an Deezer getraut? Oder ist das einfach unmöglich?
Ich habe es irgendwie noch nicht hinbekommen ...

**Comment by NoTrace-Kai** [comment]
Habe hier noch mal die verfügbaren "Bibi und Tina" Folgen und "Bibi Blocksberg" Folgen von Amazon-Music
zusammengefügt.

[Bibi und Tina und Bibi Blocksberg Amazon.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/5989699/Bibi.und.Tina.und.Bibi.Blocksberg.Amazon.txt)

**Reply by Ozelotschwein** [reply]
Vielen Dank!  zur info die Coverlinks funktionieren nicht

**Reply by NoTrace-Kai** [reply]
Bei mir schon ?!

**Reply by Ozelotschwein** [reply]
https://user-images.githubusercontent.com/77623505/108087428-b7ae3a80-7077-11eb-8dc4-f487be062ef6.png

**Reply by NoTrace-Kai** [reply]
<img width="1595" alt="Bildschirmfoto 2021-02-16 um 16 56 51" src="https://user-images.githubusercontent.com/34300928/108087744-065bd480-7078-11eb-9924-e1c01dfb172a.png">
<img width="1595" alt="Bildschirmfoto 2021-02-16 um 16 56 40" src="https://user-images.githubusercontent.com/34300928/108087771-0a87f200-7078-11eb-859c-4ac72a37772a.png">

**Reply by Ozelotschwein** [reply]
Also die Coverlinks funktionieren nur nicht im Safari-Browser! warum auch immer....

**Reply by NoTrace-Kai** [reply]
Nabend ...

So, ich habe nun nochmal alle Bibi und Tina Folgen und alle Bibi Blocksberg zusammengepackt.
Sind aber eine Mischung aus Apple-Music und Amazon-Music.

CU Kai

[Bibi und Tina - Bibi Blocksberg.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/6045177/Bibi.und.Tina.-.Bibi.Blocksberg.txt)

**Comment by NoTrace-Kai** [comment]
Ich habe die Screenshots auch im Safari Browser gemacht ... seltsam ...

**Comment by Hasenpups** [comment]
Ich könnte auch noch Hörbücher für Apple Music beisteuern:

Ninjago
Bob der Baumeister
Dinotrux
Robin Hood

Einfach den Inhalt aus Spalte E kopieren.

[Itunes - Hörbücher.xlsx](https://github.com/Thyraz/Sonos-Kids-Controller/files/7120371/Itunes.-.Horbucher.xlsx)

**Comment by WhileDekker** [comment]
[Der tierische Kindergarten.Amazon.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/8256567/Der.tierische.Kindergarten.Amazon.txt)

[Fabi Fuchs Hörspiele.Amazon.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/8256568/Fabi.Fuchs.Horspiele.Amazon.txt)

**Comment by Monox83** [comment]
[Octonauten.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/14311596/Octonauten.txt) Apple Musik
[Gabbys Dollhause.txt](https://github.com/Thyraz/Sonos-Kids-Controller/files/14311598/Gabbys.Dollhause.txt) Amazon Musik
